usecases/api/ma_mdt.go

```diff
@@ -0,0 +1,24 @@
+package api
+
+import (
+	"github.com/enbility/eebus-go/api"
+	spineapi "github.com/enbility/spine-go/api"
+)
+
+// Actor: Monitoring Appliance
+// UseCase: Monitoring of DHW Temperature
+type MaMDTInterface interface {
+	api.UseCaseInterface
+
+	// Scenario 1
+
+	// return the momentary temperature of the domestic hot water circuit
+	//
+	// parameters:
+	//   - entity: the entity of the device (e.g. DHWCircuit)
+	//
+	// possible errors:
+	//   - ErrDataNotAvailable if no such limit is (yet) available
+	//   - and others
+	Temperature(entity spineapi.EntityRemoteInterface) (float64, error)
```

Comment:
We also need to think about how we want to handle different units. According to the UC spec, the measurement can be reported in different units. I talked to Andreas and we said it makes sense to let the user request the temperatures in different units, so this interface needs to be adapted.